### PR TITLE
fix: biw unpublish and physics update

### LINKED ISSUE #1712: BIW: Unable to unpublish scenes
You can't unpublish scenes because it says the following error

### LINKED ISSUE #1945: BIW Physics are not working well
For some reason, physics are not working well with BIW, we have workaround it but it has some flaws, we need to investigate what  is happening

### PR DESCRIPTION
**Pull request by Suduck**
## What does this PR change?

This PR fixes #1712. It enables the unpublish from the builder in world again
This PR fixes #1945. The physics are now sync correctly when we end transforming items

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/biw-unpublish&kernel-branch=feat/biw-invalidate-scene-info)
2. Test that you can unpublish an scene

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: unity-renderer/Assets/DCLPlugins/BuilderInWorld/Publisher/Scripts/Publisher.cs [hunk 1]
```diff
@@ -128,6 +135,74 @@ public void StartPublish(IBuilderScene scene)
...
+        {
+            type = TYPE.UNPUBLISH;
+            
+            var manifest = BIWUtils.CreateEmptyDefaultBuilderManifest(size, BIWUtils.Vector2INTToString(coords));
+            
+            CatalystSceneEntityMetadata sceneJson = new CatalystSceneEntityMetadata();
+
+            //Display info
+            sceneJson.display = new CatalystSceneEntityMetadata.Display();
+            sceneJson.display.title = "Empty";
```
Comment (pravusjif) [inline unity-renderer/Assets/DCLPlugins/BuilderInWorld/Publisher/Scripts/Publisher.cs:148]: this string could be in a const

File: unity-renderer/Assets/DCLPlugins/BuilderInWorld/Publisher/Scripts/Publisher.cs [hunk 2]
```diff
@@ -128,6 +135,74 @@ public void StartPublish(IBuilderScene scene)
...
+            //Scenes
+            sceneJson.scene = new CatalystSceneEntityMetadata.Scene();
+
+            //  Base Parcels 
+            string baseParcels = BIWUtils.Vector2INTToString(coords);
+            sceneJson.scene.@base = baseParcels;
+
+            //  All parcels 
+            string[] parcels = new string[size.x*size.y];
+            int cont = 0;
```
Comment (pravusjif) [inline unity-renderer/Assets/DCLPlugins/BuilderInWorld/Publisher/Scripts/Publisher.cs:159]: better call the variable `count` right?

**Review by pravusjif — APPROVED** [review]

File: unity-renderer/Assets/DCLPlugins/BuilderInWorld/Publisher/Scripts/Publisher.cs (hunk 2 above)
Comment (Suduck) [reply unity-renderer/Assets/DCLPlugins/BuilderInWorld/Publisher/Scripts/Publisher.cs]: done

File: unity-renderer/Assets/DCLPlugins/BuilderInWorld/Publisher/Scripts/Publisher.cs (hunk 1 above)
Comment (Suduck) [reply unity-renderer/Assets/DCLPlugins/BuilderInWorld/Publisher/Scripts/Publisher.cs]: done

**Pull request merged**